parastell.py

```diff
@@ -202,20 +203,92 @@ def native_export():
 
     dir = Path(export['dir'])
     filename = Path(export['h5m_filename'])
+    
+    def merge_layer_surfaces(): # assumes that components dict is ordered from inside to out
+        """Merge surfaces based on surface ids rather than imprinting/merging all
+        """
+        def orient_spline_surfaces(volume_id):
+            """Return the inner and outer surface ids for a given volume id
+            """
+            surfaces = cubit.get_relatives('volume', volume_id, 'surface')
+                
+            splineSurfaces = []
+
+            for surf in surfaces:
+                if cubit.get_surface_type(surf) == 'spline surface':
+                    splineSurfaces.append(surf)
+
+            # check if this is the plasm
+            if len(splineSurfaces) == 1:
+                outerSurface = splineSurfaces[0]
+                innerSurface = None
+            
+            else:
+                # the outer surface bounding box will have the larger max xy value
+                if cubit.get_bounding_box('surface', splineSurfaces[1])[4] > cubit.get_bounding_box('surface',splineSurfaces[0])[4]:
+                    outerSurface = splineSurfaces[1]
+                    innerSurface = splineSurfaces[0]
+
+                else:
+                    outerSurface = splineSurfaces[0]
+                    innerSurface = splineSurfaces[1]
+                
+            return innerSurface, outerSurface
```

Comment:
I don't think there's a good reason for this to be nested in the other function.  I think we should only do that when there is a driving reason to do so...

Reply:
Should I bump `legacy_export` and `native_export` out to be their own functions as well then? Or is it ok to have these local to `cubit_export`?  I appreciate input on good style/practices since I don't have much sense of those. I had originally nested all these functions to level where they would get used to limit there scope

Reply:
The export ones make slightly more sense to me than this one.  Not really sure why.  I guess there is no use case for calling the export functions from outside the `cubit_export` function, but I can imagine that this function could have such a use case.